libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -282,6 +282,12 @@ bool AC_Fence::pre_arm_check(const char* &fail_msg) const
 {
     fail_msg = nullptr;
 
+    // if fences are enabled but none selected fail pre-arm check
+    if ( enabled() && !present() ) {
```

Comment:
can you remove the little space between "(" and "enabled()"?

Reply:
done